plugins/octopus-deploy/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@backstage/plugin-octopus-deploy",
-  "version": "0.1.2-next.0",
+  "version": "0.1.1-next.1",
```

Comment:
Maybe this is a rebase problem? Should revert this change

Reply:
Probably. Restored it.